src/content/docs/more-integrations/cloudfoundry-integrations/vmware-tanzu-service-broker-integration.mdx

```diff
@@ -0,0 +1,423 @@
+
+import infrastructureVmwareTanzuImport4 from 'images/vmware-service-broker-import-4.png'
+
+import infrastructureVmwareTanzuImport5 from 'images/vmware-service-broker-import-5.png'
+
+import infrastructureVmwareTanzuImport6 from 'images/vmware-service-broker-import-6.png'
+
+import infrastructureVmwareTanzuImport7 from 'images/vmware-service-broker-import-7.png'
+
+import infrastructureVmwareTanzuImport8 from 'images/vmware-service-broker-import-8.png'
+
+import infrastructureVmwareTanzuImport9 from 'images/vmware-service-broker-import-9.png'
+
+import infrastructureVmwareTanzuImport10 from 'images/vmware-service-broker-import-10.png'
+
+import infrastructureVmwareTanzuImport11 from 'images/vmware-service-broker-import-11.png'
+
+import infrastructureVmwareTanzuImport12 from 'images/vmware-service-broker-import-12.png'
+
+import infrastructureVmwareTanzuImport13 from 'images/vmware-service-broker-import-13.png'
+
+import infrastructureVmwareTanzuImport14 from 'images/vmware-service-broker-import-14.png'
+
+import infrastructureVmwareTanzuImport15 from 'images/vmware-service-broker-import-15.png'
+
+import infrastructureVmwareTanzuImport16 from 'images/vmware-service-broker-import-16.png'
+
+import infrastructureVmwareTanzuImport17 from 'images/vmware-service-broker-import-17.png'
+
+import infrastructureVmwareTanzuImport18 from 'images/vmware-service-broker-import-18.png'
+
+New Relic Service Broker for VMware Tanzu enables you to use one or more New Relic accounts and is deployed as a Java app on VMware Tanzu. A service broker allows Cloud Foundry apps to bind to services and consume the services easily from the Apps Manager or from the command line.
+
+The broker exposes the New Relic service on the Marketplace and allows users to directly create a service instance and bind it to their apps either from Apps Manager or from the command line.
+
+The New Relic Service Broker for VMware Tanzu tile installs the New Relic Service Broker as an app, registers it as a service broker on VMware Tanzu, and exposes its service plans on the Marketplace. Each service plan is associated with an existing New Relic account, which is configured during the tile setup.
+
+Selecting a plan binds your app with the New Relic agent, and the agent starts reporting to the New Relic account which is associated with the selected plan. This makes the installation and subsequent use of New Relic on your VMware Tanzu apps easier and more straightforward.
+
+## Prerequisites [#prereqs]
+
+New Relic Nozzle for VMware Tanzu has the following requirements:
+
+* Service broker v1.12.18 and later
+* For service brokers on v1.12.12 and earlier, the unique GUIDs for plans were calculated differently. For these plans to not break compatibility, the GUIDs must be the same as before. The migration script preserves the plan GUIDs for existing plans in the plan collection for v1.12.12 and earlier.
+
+<Collapser
```

Comment:
This seems like an alternate path for users on a different version. It's unclear to me what they're trying to achieve by running this script, but for now I've nested the info in a collapser to "chunk" this / differentiate it from the rest of the prereqs